EIPS/eip-2935.md

```diff
@@ -158,37 +153,43 @@ A special synthetic address is generated by working backwards from the desired d
   "v": "0x1b",
   "r": "0x539",
   "s": "0x1b9b6eb1f0",
-  "hash": "7ba81426bfa88a2cf4ea5c9abbbe83619505acd1173bc8450f93cf17cde3784b",
+  "hash": "7ba81426bfa88a2cf4ea5c9abbbe83619505acd1173bc8450f93cf17cde3784b"
 }
 ```
 
 Note, the input in the transaction has a simple constructor prefixing the desired runtime code.
 
 The sender of the transaction can be calculated as `0xa4690f0ed0d089faa1e0ad94c8f1b4a2fd4b0734`. The address of the first contract deployed from the account is `rlp([sender, 0])` which equals `0x25a219378dad9b3503c8268c9ca836a52427a4fb`. This is how `HISTORY_STORAGE_ADDRESS` is determined. Although this style of contract creation is not tied to any specific initcode like create2 is, the synthetic address is cryptographically bound to the input data of the transaction (e.g. the initcode).
 
+Some fork (blocks where `block.timestamp >= FORK_TIMESTAMP`) activation scenarios:
 
-Some activation scenarios:
+- For the fork to be activated at genesis, no history is written to the genesis state, and at the start of block `1`, genesis hash will be written as a normal operation to slot `0`.
+- for fork activation at block `1`, only genesis hash will be written at slot `0`.
+- for fork activation at block `32`, block `31`'s hash will be written to slot `31`. Additional history for block `0..30` hashes will continue to be served, but will not be backfilled into storage.
+- for fork activation at block `10000`, block `9999`'s hash will be written to slot `9999`. Additional history for the the `BLOCKHASH_OLD_WINDOW` (`9744..9998`) will continue to be served, but will not be backfilled into storage. History for the blocks preceding the `HISTORY_SERVE_WINDOW` (`1808..9998`) will not be served, and will not be backfilled into storage. At block `20000`, the BLOCKHASH opcode would continue serving history for the `BLOCKHASH_OLD_WINDOW` (`19744..19999`) and would return 0 for blocks before the `BLOCKHASH_OLD_WINDOW`. The blocks from `HISTORY_SERVE_WINDOW` would be served from state for blocks `11808..19999`. Blocks prior to the `HISTORY_SERVE_WINDOW` would return 0.
```

Comment:
this should be broken into 2 examples and "will not be served" should be replaced by "will return 0" for clarity